### PR TITLE
Add people endpoint support

### PR DESCRIPTION
**Pull request by jcomello**

### CODE REVIEW

**Comment by jcomello** [comment]
[ch12190](https://app.clubhouse.io/nexaas/story/12190/5-alterar-entidade-payer-para-person)

File: lib/cobrato/entities/payer.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 module Cobrato
   module Entities
     class Payer < Base
+      puts "Warning: Payer entity is deprecated. Please use Person entity instead."
```
Comment (RafaelMCarvalho) [inline lib/cobrato/entities/payer.rb]: I suggest to warn only when someone use the methods. See [regress_cnab.rb:9](https://github.com/myfreecomm/cobrato-client-ruby/blob/master/lib/cobrato/resources/regress_cnab.rb#L9)

File: lib/cobrato/resources/payer.rb [hunk 2]
```diff
@@ -1,7 +1,14 @@
 module Cobrato
   module Resources
     class Payer < Base
+      puts "Warning: Payers endpoint is deprecated. Please use People endpoint instead."
```
Comment (RafaelMCarvalho) [inline lib/cobrato/resources/payer.rb]: I suggest to warn only when someone use the methods. See [regress_cnab.rb:9](https://github.com/myfreecomm/cobrato-client-ruby/blob/master/lib/cobrato/resources/regress_cnab.rb#L9)

File: spec/cobrato/resources/payer_spec.rb [hunk 3]
```diff
@@ -62,7 +62,7 @@
         payer = subject.create(params)
         expect(payer).to be_a(entity_klass)
         expect(payer.name).to eq(params['name'])
-        expect(payer.id).to eq(8)
+        expect(payer.id).to eq(4016)
```
Comment (RafaelMCarvalho) [inline spec/cobrato/resources/payer_spec.rb]: We can use `person` here

File: lib/cobrato/entities/payer.rb (hunk 1 above)
Comment (RafaelMCarvalho) [reply lib/cobrato/entities/payer.rb]: Corrigido em https://github.com/myfreecomm/cobrato-client-ruby/pull/64/commits/13e7c6c3b7c53a560df975589e984891df5c46c1

**Comment by jcomello** [comment]
@RafaelMCarvalho Fixed review at https://github.com/myfreecomm/cobrato-client-ruby/pull/64/commits/cfff506607e1c80830911414259c90f74849106c

**Comment by RafaelMCarvalho** [comment]
@jcomello Beleza, só faltou citar que o endpoint `payers` está deprecated.

**Comment by jcomello** [comment]
Faltou também adicionar o método `people` na classe client.

**Pull request merged**